XRTK-Core/Packages/com.xrtk.core/Definitions/InputSimulationSystem/MixedRealityInputSimulationSystemProfile.cs

```diff
@@ -0,0 +1,186 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using XRTK.Definitions.Devices;
+using XRTK.Definitions.Utilities;
+using XRTK.Services.InputSimulationSystem;
+
+namespace XRTK.Definitions.InputSimulationSystem
```

Comment:
Can we change the namespace to 
```
XRTK.Definitions.InputSystem.Simulation
```
Makes more sense that way, as the input system is required for simulation, it's not a replacement